FestivalPlanner/films/views.py

```diff
@@ -757,25 +767,30 @@ def get_context_data(self, *, object_list=None, **kwargs):
             'total_film_count': self.total_film_count,
             'judged_href_filter': self.judged_filter.get_href_filter(session),
             'judged_filter_action': self.judged_filter.action(session),
+            'festival_href_filter': self.festival_filter.get_href_filter(session),
+            'festival_filter_action': self.festival_filter.action(session),
+            'festival_filter_label': self.festival_filter.label(session),
             'unexpected_errors': self.unexpected_errors,
         }
         context = add_base_context(self.request, super_context | new_context)
         return context
 
-    def get_reviewer_rows(self, reviewers):
+    def _get_reviewer_rows(self, reviewers):
+        pr_debug('start', with_time=True)
         reviewer_rows = []
         for reviewer in reviewers:
-            row = self.get_reviewer_row(reviewer)
+            row = self._get_reviewer_row(reviewer)
             if row['display_reviewer']:
                 reviewer_rows.append(row)
+        pr_debug('done', with_time=True)
```

Comment:
Please consider to remove the debug statements.
When you decided to speed up things, they will be useful, not now.